### PR TITLE
chore: Package move `com.eppo.sdk` -> `cloud.eppo`

### PR DESCRIPTION
**Pull request by felipecsl**
So this is consistent with Android

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ java {
 }
 
 group = 'cloud.eppo'
-version = '3.1.1-SNAPSHOT'
+version = '4.0.0-SNAPSHOT'
```
Comment (felipecsl) [inline build.gradle:14]: bumped major version since it's a breaking change

File: src/main/java/cloud/eppo/AppDetails.java [hunk 2]
```diff
@@ -1,25 +1,25 @@
-package com.eppo.sdk.helpers;
+package cloud.eppo;
```
Comment (typotter) [inline src/main/java/cloud/eppo/AppDetails.java:1]: on a conceptual level, is the `android` SDK a sub-implementation of the java SDK or are the `android` **and** `java-server` SDKs both parallel flavours of a `jvm` SDK?

**Review by typotter — APPROVED** [review]
Approved with just a thought about the package naming for you to consider (could be an overthought; I trust your judgement).

File: src/main/java/cloud/eppo/EppoClient.java [hunk 3]
```diff
@@ -53,7 +50,10 @@ private EppoClient(
         isGracefulModel,
         false,
         true,
-        null);
+        null,
+        null, // TODO assignmentCache,
```
[the remark this answers is not in the document]
Comment (felipecsl) [reply src/main/java/cloud/eppo/EppoClient.java:54]: cool, I'll rebase with that

File: src/main/java/cloud/eppo/AppDetails.java (hunk 2 above)
Comment (felipecsl) [reply src/main/java/cloud/eppo/AppDetails.java:1]: both are separate, completely individual libraries. they can share the same package names though, AFAIK there are no constraints preventing that

**Pull request merged**